README.md

```diff
@@ -295,42 +295,48 @@ npm install
 npm run build
 ```
 
-### Download and install needed artifacts (`Optional`)
+### Download and install needed artifacts (optional)
 
-For easier and faster setup of local environment, you can run:
+For an easier and faster setup of your local environment, run:
 
 ```bash
-❯ node dist/cli.js setup <binaries>
+node dist/cli.js setup <binaries>
+```
 
-Setup is meant for downloading and making everything ready for dev environment of ZombieNet;
+This allows to use the `setup` script, making everything ready for a ZombieNet dev environment.
 
 You can use the following arguments:
 
---help shows this message;
---binaries or -b: the binaries that you want to be downloaded and installed during the setup, provided in a row without any separators;
-	possible options: 'polkadot', 'polkadot-parachain'
-	example: node dist/cli.js setup polkadot polkadot-parachain
+`--help` shows the different options and commands for using the Zombienet CLI.
+`--binaries` or `-b`: enables providing the binaries that you want to be downloaded and installed during the setup. Possible options: `polkadot`, `polkadot-parachain`.
+
+For example:
+
+```bash
+node dist/cli.js setup polkadot polkadot-parachain
 ```
-> Note: If you are using MacOS. Please, clone the polkadot repo (https://github.com/paritytech/polkadot) and run it locally. At the moment there is no `polkadot` binary for MacOs.
 
-Script above will retrieve the binaries provided and try to download and prepare those binaries for
-usage. At the end of the download, script will provide a command to run in your local environment in
-order to add the directory where the binaries were downloaded in your $PATH var. E.g.:
+> Note: If you are using macOS please clone the [Polkadot repo](https://github.com/paritytech/polkadot) and run it locally. At the moment there is no `polkadot` binary for MacOs.
+
+The command above will retrieve the binaries provided and try to download and prepare those binaries for usage. 
+At the end of the download, the `setup` script will provide a command to run in your local environment in order to add the directory where the binaries were downloaded in your $PATH var, for example:
 
 ```bash
-Please add the dir to your $PATH by running the command: export PATH=/home/<user>/zombienet/dist:$PATH
+Please add the dir to your $PATH by running the command: export PATH=/home/<user>/current_directory:$PATH
```

Comment:
Agreeing with @pepoviola here.
The `dist` dir should always be there. We did that, so that the "cleanup" process of a user will be easier - meaning that once dist dir is removed, then also the downloaded files will be removed

```suggestion
Please add the dir to your $PATH by running the command: export PATH=/home/<user>/<current_directory>/dist:$PATH
```